### PR TITLE
fix: [ST-2310] Redirection only for get request method

### PR DESCRIPTION
**Pull request by Soichiro725**
### Purpose/goal of Pull Request (Issue #)
https://wovnio.atlassian.net/browse/ST-2310

Wovnrb redirects to cookie lang URL only for GET request method.

### CODE REVIEW

File: lib/wovnrb.rb [hunk 1]
```diff
@@ -37,7 +37,9 @@ def call(env)
 
       cookie_lang = Rack::Request.new(env).cookies['wovn_selected_lang']
       request_lang = headers.lang_code
-      if @store.settings['use_cookie_lang'] && cookie_lang.present? && request_lang != cookie_lang && request_lang == @store.default_lang
+      is_get_request = Rack::Request.new(env).get?
```
Comment (bstewart00) [inline lib/wovnrb.rb:40]: time to make `request = Rack::Request.new(env)` variable at the top

File: lib/wovnrb.rb [hunk 2]
```diff
@@ -37,7 +37,9 @@ def call(env)
...
+
+      if @store.settings['use_cookie_lang'] && cookie_lang.present? && request_lang != cookie_lang && request_lang == @store.default_lang && is_get_request
```
Comment (bstewart00) [inline lib/wovnrb.rb:42]: the same bug would have existed for the redirect on line 48 too (in all libraries)

File: lib/wovnrb.rb (hunk 1 above)
Comment (Soichiro725) [reply lib/wovnrb.rb]: Fixed https://github.com/WOVNio/wovnrb/pull/238/commits/a55f6e07fa2e7346ff294e0332fe310947a135f9

File: lib/wovnrb.rb (hunk 2 above)
Comment (Soichiro725) [reply lib/wovnrb.rb:43]: Fixed: https://github.com/WOVNio/wovnrb/pull/238/commits/a55f6e07fa2e7346ff294e0332fe310947a135f9

**Review by bryanzerrudo — APPROVED** [review]

**Review by bstewart00 — APPROVED** [review]

**Pull request merged**